vega_sim/scenario/comprehensive_market/scenario.py

```diff
@@ -321,6 +321,7 @@ def set_up_background_market(
     def run_iteration(
         self,
         vega: VegaServiceNull,
+        network: Optional[str] = None,
```

Comment:
These should be `Optional[Network]`s right?

Reply:
Thanks yep have changed throughout